goal_src/jak2/engine/draw/drawable.gc

```diff
@@ -1205,9 +1205,10 @@
                     ;; PC port note : we ALWAYS disable the envmap hack when a process-drawable has warp effect enabled
                     (when (or (= lod-idx (-> dc cur-lod)) (logtest? (-> dc status) (draw-control-status lod-set)))
                         (protect ((-> *pc-settings* force-envmap?))
-                          (dotimes (eff-i (-> dc mgeo header effect-count))
-                            (if (logtest? (effect-bits cross-fade) (-> dc mgeo effect eff-i effect-bits))
-                              (false! (-> *pc-settings* force-envmap?))))
+                          (when (not (movie?))
```

Comment:
is this for the baron throne room cutscene, when you can see the metal head hologram from far away?

Reply:
yes